src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/IntegerCopyNumberTransitionMatrixCollectionReader.java

```diff
@@ -0,0 +1,118 @@
+package org.broadinstitute.hellbender.tools.exome.germlinehmm;
+
+import com.google.cloud.dataflow.sdk.repackaged.com.google.common.collect.Sets;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableUtils;
+
+import javax.annotation.Nonnull;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class reads a collection of copy number transition matrices. The collection is a tab-separated table
```

Comment:
@samuelklee @davidbenjamin good suggestions -- I made the reader a private static inner class and updated the javadocs.